From e955808b648c4546a0e084dc0a6bf60149c33f79 Mon Sep 17 00:00:00 2001
From: Ramanathan Choodamani <quic_rchoodam@quicinc.com>
Date: Tue, 16 Jul 2024 17:24:49 -0700
Subject: [PATCH 2/2] wifi: mac80211: Remove fast_xmit override for mesh fast
 xmit

This change will make use of fast_xmit flag for mesh mode
which will utilise the fast_tx path in the driver

Patch dependency: none

Signed-off-by: Ramanathan Choodamani <quic_rchoodam@quicinc.com>
---
 net/mac80211/tx.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/net/mac80211/tx.c
+++ b/net/mac80211/tx.c
@@ -4631,9 +4631,6 @@ netdev_tx_t ieee80211_subif_start_xmit(s
 #ifdef CPTCFG_MAC80211_NSS_SUPPORT
 	ieee80211_xmit_nss_fixup(skb, dev);
 #endif
-#ifdef CPTCFG_MAC80211_SFE_SUPPORT
-	skb->fast_xmit = 0;
-#endif
 
 	if (likely(!is_multicast_ether_addr(eth->h_dest)))
 		goto normal;
